frontend/www/js/omegaup/arena/arena.ts

```diff
@@ -291,6 +292,8 @@ export class Arena {
     problemAlias: string;
   };
 
+  problemSettingsSummary: (Vue & { problem: Problem }) | null = null;
```

Comment:
```suggestion
  problemSettingsSummary: (Vue & { problem: types.ProblemDetails }) | null = null;
```

en el `.vue` se está declarando como `types.ProblemDetails`, así que aquí hay que hacer lo mismo.

Reply:
Listo. Tuve que crear otra anotación con los elementos que se requieren, para no tener que meter tantos opconales